src/react-app/src/configs/i18n.ts

```diff
@@ -0,0 +1,23 @@
+import i18n from 'i18next';
```

Comment:
This file configures i18next. Tells it where to load translations, what language to fallback to, adds some debug logging, and configures a browser language detector.

Reply:
Dope.